azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/database/DatabaseService.java

```diff
@@ -115,18 +117,20 @@ private SecretKey decodeBase64Key(String encryptionKeyBase64) {
   }
 
   private OutputStream encryptIntoOutputStream(OutputStream origin, String encryptionKeyBase64)
-      throws NoSuchPaddingException, NoSuchAlgorithmException, InvalidKeyException {
+      throws NoSuchPaddingException, NoSuchAlgorithmException, InvalidKeyException,
+          InvalidAlgorithmParameterException {
     SecretKey encryptionKey = decodeBase64Key(encryptionKeyBase64);
-    Cipher c = Cipher.getInstance("AES");
-    c.init(Cipher.ENCRYPT_MODE, encryptionKey);
+    Cipher c = Cipher.getInstance("AES/GCM/NoPadding");
```

Comment:
Using `NoPadding` as suggested here: https://stackoverflow.com/questions/31248777/can-pkcs5padding-be-in-aes-gcm-mode